Src/Extern/HYPRE/AMReX_Habec_3D_K.H

```diff
@@ -914,14 +913,16 @@ void habec_ijmat_eb (GpuArray<Real,27>& sten, Array4<Int> const& ncols,
         }
 
         if (beb) {
-            Real anorm = std::sqrt((apx(i,j,k) - apx(i+1,j,k))*(apx(i,j,k) - apx(i+1,j,k))
-                                 + (apy(i,j,k) - apy(i,j+1,k))*(apy(i,j,k) - apy(i,j+1,k))
-                                 + (apz(i,j,k) - apz(i,j,k+1))*(apz(i,j,k) - apz(i,j,k+1)));
+
+            Real dapx = (apx(i,j,k) - apx(i+1,j,k))*dx[1]*dx[2];
+            Real dapy = (apy(i,j,k) - apy(i,j+1,k))*dx[0]*dx[2];
+            Real dapz = (apz(i,j,k) - apz(i,j,k+1))*dx[1]*dx[2];
```

Comment:
```suggestion
            Real dapx = (apx(i,j,k) - apx(i+1,j,k));
            Real dapy = (apy(i,j,k) - apy(i,j+1,k));
            Real dapz = (apz(i,j,k) - apz(i,j,k+1));
```
Since 3D has not been implemented, could we just remove those `dx`s?

Reply:
Never mind. Maybe we could keep them for consistence because similar changes have been made in MLMG.

Reply:
I just changed my mind again.  I think we should revert all the changes in 3D to be safe.

Reply:
I reverted the changes in 3D (except for the changed interfaces).